src/hotspot/share/prims/whitebox.cpp

```diff
@@ -2497,6 +2504,12 @@ WB_ENTRY(jint, WB_ValidateCgroup(JNIEnv* env,
   return ret;
 WB_END
 
+// Physical cpus of the host machine (including containers), Linux only.
```

Comment:
Isn't the comment a bit misleading ?  From what I see , ` os::Linux::active_processor_count()` can use various mechanisms to get number of processor info, if it uses https://linux.die.net/man/2/sched_getaffinity it gives the 'set of CPUs on which it is eligible to run.'   That might be different from what the host has.

Reply:
Yes. See #20768 for an attempt to unify it. I'll change the comment with the update that I have for nested hierarchies. Thanks!

Reply:
I've changed the comment.